history-of-the-quakers-v4/original/01-book-6-chapter-14.adoc

```diff
@@ -48,13 +59,19 @@ to the glory of God, and the benefit of these nations.
 May the King of Kings make thy reign long and glorious,
 to which temporal blessing we shall pray for thy eternal happiness.
 
+[.signed-section-context-close]
```

Comment:
Not sure if this is the best way for this.

Reply:
i think it seems good.